mod-notes/mod-notes.postman_collection.json

```diff
@@ -8215,63 +8007,560 @@
+									"            'Content-Type': 'application/json'",
+									"        },",
+									"         body: {",
+									"           mode: 'raw',",
+									"            raw:  JSON.stringify({",
+									"                \"module\": \"NOTES\",",
+									"                \"configName\": \"api_access\",",
+									"                \"code\": \"note.types.number.limit\",",
+									"                \"value\": pm.environment.get(\"config-value-three\")",
+									"            })",
+									"        }",
+									"    }, function(err, res) {",
+									"        pm.test(\"Status no content\", function() {",
+									"        pm.expect(res.status).eq(\"No Content\");",
+									"        });",
+									"    });",
+									"    }",
+									"    postman.setNextRequest(\"POST create note-type for tests\");",
+									"}",
+									"",
+									"",
+									"",
 									""
 								],
 								"type": "text/javascript"
 							}
 						},
 						{
-							"listen": "test",
+							"listen": "prerequest",
 							"script": {
-								"id": "d7912ab7-5630-4a3c-9608-c03a42bed5fe",
+								"id": "d665ae85-57e5-40c5-a197-48a2b098fc0a",
 								"exec": [
-									"//Check that status is 204",
-									"pm.test(\"Status is 204\", function () {",
-									"    pm.response.to.have.status(204);",
-									"});",
-									"",
-									"pm.environment.unset(\"noteId\");",
-									"pm.environment.unset(\"noteContent\");",
-									"pm.environment.unset(\"noteTitle\");"
+									"pm.environment.set(\"config-value-three\",3)"
 								],
 								"type": "text/javascript"
 							}
 						}
 					],
 					"request": {
-						"method": "DELETE",
+						"method": "GET",
 						"header": [
 							{
-								"key": "X-Okapi-Token",
-								"type": "text",
-								"value": "{{xokapitoken}}"
-							},
-							{
-								"key": "X-Okapi-Tenant",
-								"type": "text",
+								"key": "x-okapi-tenant",
 								"value": "{{xokapitenant}}"
 							},
 							{
-								"key": "Content-Type",
-								"name": "Content-Type",
-								"type": "text",
-								"value": "application/json"
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
 							}
 						],
-						"body": {
-							"mode": "raw",
-							"raw": ""
-						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/configurations/entries?query=(module==NOTES and configName==api_access and code==note.types.number.limit)",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"configurations",
+								"entries"
+							],
+							"query": [
+								{
+									"key": "query",
+									"value": "(module==NOTES and configName==api_access and code==note.types.number.limit)"
+								}
+							]
+						},
+						"description": "Check if rm api url has already been configured for test user.  If so, do not want to overwrite existing configuration settings"
+					},
```

Comment:
Really minor comment -- could description just reference note type limit

Reply:
Sorry, my bad. I'll fix it.